src/postman/operation.ts

```diff
@@ -77,7 +81,12 @@ function transformItem(item: Item, securitySchemes: PostmanSecurityScheme[]): IH
     path: getPath(item.request.url),
     summary: item.name,
     request,
-    responses: item.responses.map(transformResponse),
+    responses: item.responses
+      .all()
+      .reduce<IHttpOperationResponse[]>(
+        (merged, response) => mergeResponses(merged, [transformResponse(response)]),
+        [],
+      ) as IHttpOperation['responses'],
```

Comment:
Why is such cast required? What are you getting back?

Reply:
I get `NonEmptyArray` in return. Actually I thought today that I could change `mergeLists` to operate on Array types. So this cast would not be needed.

Reply:
More or less like that (I'm not sure whether this is good way to extract item type from array type).
```
function mergeLists<T>(compare: (o1: T[0], o2: T[0]) => boolean, merge: (o1: T[0], o2: T[0]) => T[0]) {
```